doc/developer/debugging.rst

```diff
@@ -38,12 +43,11 @@ Open the application at the following URL: ``https://localhost:8484/dev/<interfa
 Pyramid debugtoolbar
 --------------------
 
-If it is not already done, copy the file ``docker-compose.override.sample.yaml`` to
-``docker-compose.override.yaml``.
+With the default ``docker-compose.override.yaml`` you should have the debug toolbar.
```

Comment:
Suggestion:
With the default ``docker-compose.override.yaml``, the debug toolbar should appear.